cmd/goal/account.go

```diff
@@ -579,6 +610,23 @@ var addParticipationKeyCmd = &cobra.Command{
 	},
 }
 
+var installParticipationKeyCmd = &cobra.Command{
+	Use:   "installpartkey",
+	Short: "Install a participation key",
+	Long:  `Install a participation key`,
+	Args:  validateNoPosArgsFn,
+	Run: func(cmd *cobra.Command, args []string) {
+		dataDir := ensureSingleDataDir()
+
+		client := ensureFullClient(dataDir)
```

Comment:
We don't need a full client for this do we (just algod)?

Reply:
Yup, sloppy copy-pasting of libgoal boilerplate on my part.